docs/Advanced-Usage/Debugging-and-Profiling-on-FPGA/TracerV.rst

```diff
@@ -0,0 +1,280 @@
+.. _tracerv:
+
+Capturing RISC-V Instruction Traces with TracerV
+==================================================
+
+FireSim can provide a cycle-by-cycle trace of a target CPU's architectural
```

Comment:
it needs to be more clear that TracerV no longer exports all of these fields by default.

Reply:
this is noted in the next section about building